decidim-core/app/cells/decidim/content_blocks/highlighted_content_banner/show.erb

```diff
@@ -7,7 +7,7 @@
           <%= translated_attribute current_organization.highlighted_content_banner_title %>
         </h1>
         <span class="text-highlight">
-          <%= decidim_sanitize_editor translated_attribute current_organization.highlighted_content_banner_short_description %>
+          <%= decidim_sanitize translated_attribute current_organization.highlighted_content_banner_short_description %>
```

Comment:
This was introduced to support lists within lists, with this last change this breaks: 

![image](https://user-images.githubusercontent.com/717367/151938813-3a602c68-c565-4eb1-902e-182c3b3e5003.png)

Let's change the span to a div and leave the decidim_sanitize_editor

It should look like this:
```ruby
        <div class="text-highlight">
          <%= decidim_sanitize_editor translated_attribute current_organization.highlighted_content_banner_short_description %>
        </div>
```

(GitHub UI doesn't allow me to add this as a suggestion :S)